CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ For details about compatibility between different releases, see the **Commitment
 ### Added
 
 - Fine grained NbTrans overrides for the dynamic ADR mode. See the `--mac-settings.adr.mode.dynamic.overrides.data-rate-[index].min-nb-trans` and `--mac-settings.adr.mode.dynamic.overrides.data-rate-[index].max-nb-trans` family of parameters.
+- Support for user's bookmarks was added to the Stack, allowing to link an user to a specific entity.
```

Comment:
Change main text to : Support for `user_bookmarks` field to the `users` table.
Please drop the CLI info since we intedn this feature to be primarily used by the console.

Reply:
changed the main text to `Support for storing user's bookmarks in the database.`, the logic of doing that instead of the suggested text is that the `user_bookmarks` is its own table on the database.

So instead of making it seem like a field on the existing `users` table I just specified that the only thing added was the support of storing bookmarks on the DB.